### DISCUSSION TITLE
Se requiere saber diseño para Frontend ? 🤔

### DISCUSSION THREAD

**Discussion by jesuscarlosv** [Launch-X-Latam/MisionFrontEnd#32] [Q&A] [answered]
Hola, mi duda es si se requiere tener nociones o ser diseñador para ser un buen Frontend? y por otra parte, si se quiere ser Fullstack orientado a Backend, que tanto se debe profundizar en Frontend? 🤔

**Comment by Cain-ARG** [comment]
Buenas, normalmente en el trabajo están los UI y UX que son los que diseñan la web y a vos como desarrollador te entregan en archivo en Figma (ósea como se tiene que ver) y tu función es lograr eso con código.

**Comment by Emilio17MC** [comment, marked as answer]
No es totalmente necesario, pues existe el rol de Diseñador Web, que es quien se encarga del diseño UI/UX. Sin embargo, si sería importante tener ciertos conocimientos, pues puede ayudarte a comprender mejor lo que el diseñador busca transmitir cuando lo estés implementando como desarrollador web.

Cómo se puede ver en el temario en la sección "1. Intro", en el tema ["3. Setup de un programador FrontEnd"](https://github.com/Launch-X-Latam/MisionFrontEnd/blob/main/01%20-%20INTRO/temario/3.-webDevSetup.md) veremos algunas de las herramientas usadas para el diseño web.
